watson_developer_cloud/text_to_speech_v1.py

```diff
@@ -12,123 +14,1198 @@
+        if voice is None:
+            raise ValueError('voice must be provided')
+        params = {'customization_id': customization_id}
+        url = '/v1/voices/{0}'.format(*self._encode_path_vars(voice))
         response = self.request(
-            method='POST', url='/v1/synthesize', stream=True, params=params,
-            json=data)
-        return response.content
+            method='GET', url=url, params=params, accept_json=True)
+        return response
+
+    def list_voices(self):
+        """
+        Retrieves all voices available for speech synthesis.
+
+        Lists information about all available voices. To see information about a specific
+        voice, use the `GET /v1/voices/{voice}` method.
 
-    def voices(self):
+        :return: A `dict` containing the `Voices` response.
+        :rtype: dict
         """
-        Returns the list of available voices to use with synthesize
+        url = '/v1/voices'
+        response = self.request(method='GET', url=url, accept_json=True)
+        return response
+
+    #########################
+    # synthesize
+    #########################
+
+    def synthesize(self,
+                   text,
+                   accept=None,
+                   accept2=None,
+                   voice=None,
+                   customization_id=None):
         """
-        return self.request(method='GET', url='/v1/voices', accept_json=True)
+        Streaming speech synthesis of the text in the body parameter.
 
-    def pronunciation(self, text, voice=None, pronunciation_format='ipa'):
+        Synthesizes text to spoken audio, returning the synthesized audio stream as an
+        array of bytes. Identical to the `GET` method but passes longer text in the body
+        of the request, not with the URL. Text size is limited to 5 KB.   If a request
+        includes invalid query parameters, the service returns a `Warnings` response
+        header that provides messages about the invalid parameters. The warning includes a
+        descriptive message and a list of invalid argument strings. For example, a message
+        such as `\"Unknown arguments:\"` or `\"Unknown url query arguments:\"` followed by
+        a list of the form `\"invalid_arg_1, invalid_arg_2.\"` The request succeeds
+        despite the warnings.
+
+        :param str text: The text to synthesize.
+        :param str accept: The requested audio format (MIME type) of the audio. You can use this header or the `accept` query parameter to specify the audio format. (For the `audio/l16` format, you can optionally specify `endianness=big-endian` or `endianness=little-endian`; the default is little endian.).
+        :param str accept2: The requested audio format (MIME type) of the audio. You can use this query parameter or the `Accept` header to specify the audio format. (For the `audio/l16` format, you can optionally specify `endianness=big-endian` or `endianness=little-endian`; the default is little endian.).
```

Comment:
We should ignore the `accept` query parameter -- it duplicates the function of the `accept` header parameter, and the header parameter is the better approach.